packages/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/PolygonController.java

```diff
@@ -46,6 +46,10 @@ public void setPoints(List<LatLng> points) {
     polygon.setPoints(points);
   }
 
+  public void setHoles(List<List<LatLng>> holes) {
```

Comment:
what happens if the holes lie outside the polygon? does the google maps api throw some error?

Reply:
Documentation from https://developers.google.com/maps/documentation/android-sdk/shapes:

If the hole intersects the outline of the polygon, the polygon will be rendered without any fill.